### ISSUE TITLE
Loading Font Weight 300 from Google Font

### ISSUE THREAD

**Issue by Internapse** [recurly/recurly-js#360]
Google Font issues have been brought up in other threads but I don't think this specific issue has been mentioned.

When trying to load the 300 font-weight of Lato, I ran into issue after issue until I inserted a pair of quotes. While Lato:300 was recognized by Google Fonts in the URL, it was not recognized as a valid font-family for CSS. Only the 400 font weight loaded when I used Lato.

- https://fonts.googleapis.com/css?family=Lato
- https://fonts.googleapis.com/css?family=Lato:300

This is not valid and does not load in RecurlyJS:

- font-family:Lato:300

In order to resolve this, I put quotes around Lato:300 and then applied a second font family of Lato. The "Lato%3A300" gets skipped over for the CSS but gets loaded into Google Fonts. The 300 and 400 fonts are successfully loaded since the quotes are ignored in the URL.

- https://fonts.googleapis.com/css?family=Lato%7CLato%3A300

The font family is then valid:

- font-family: "Lato%3A300", Lato;

To see the code fix in action:

`style: {
	all: {
		fontFamily: '"Lato%3A300",Lato',
		fontSize: '16px',
		fontStyle: 'normal',
		fontWeight: '300',
		fontColor: '#7d7975'
	}
}`

To recap, Lato:300 is not recognized as a CSS font family attribute due to the colon. But in order to get Google Fonts to load the 300 font weight, it's needed. So by using quotes around Lato:300, I'm able to load the Google Font URL and the Lato font family.

Regards,
~Scott

**Comment by chrissrogers** [comment]
Thanks for reporting this. I think we can look at this as it would be nice to support these custom weights. In the meantime, we're totally happy to accept and review pull requests. Cheers!

**Comment by kalokiston** [comment]
+1 for custom weights.

**Comment by chrissrogers** [comment]
Support for this has been added today through an update to our hosted library. Cheers!
